packages/discord.js/src/managers/GuildChannelManager.js

```diff
@@ -96,6 +96,24 @@ class GuildChannelManager extends CachedManager {
     return super.resolveId(channel);
   }
 
+  /**
+   * Adds the target channel to channel's followers.
+   * @param {TextChannelResolvable} channel The channel the follower should be added to
+   * @param {TextChannelResolvable} targetChannel The channel where the webhook should be created
+   * @param {string} [reason] Reason for creating the webhook
+   * @returns {Promise<Snowflake>} Returns created target webhook id.
+   */
+  async addFollower(channel, targetChannel, reason) {
+    const channelId = this.client.channels.resolveId(channel);
+    const targetChannelId = this.client.channels.resolveId(targetChannel);
```

Comment:
These should technically use `this.resolve()` and `this.resolveId()` since the line after throws a guild channel-related error, I guess?

```suggestion
    const channelId = this.resolveId(channel);
    const targetChannelId = this.resolveId(targetChannel);
```